assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego

```diff
@@ -18,7 +18,7 @@ CxPolicy[result] {
 		"resourceName": task.name,
 		"searchKey": sprintf("name={{%s}}.{{%s}}.require_uppercase", [task.name, modules[m]]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": "iam_password_policy.require_uppercase set and true",
+		"keyExpectedValue": "iam_password_policy.require_uppercase should be set and true",
```

Comment:
Can you improve the `keyActualValue` of this result, please?

```
"keyActualValue": "iam_password_policy.require_uppercase is undefined",
```

Reply:
Hi @rafaela-soares,
NP, 
will something like this be OK:
"keyActualValue": "iam_password_policy.require_uppercase is not set to true" or you have something else on your mind?

Reply:
sure NP